### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by upadesh** [leokhoa/laragon#405] [General]
I am currently using laragon 5 and I want to update it to latest one. How to do it?

**Comment by Pen-y-Fan** [comment]
My method is to backup all my databases (e.g. [How To Import and Export Databases in MySQL or MariaDB](https://www.digitalocean.com/community/tutorials/how-to-import-and-export-databases-in-mysql-or-mariadb) or use [heidisql SQL export](https://www.heidisql.com/help.php#sqlexport) - make sure to export the data. This isn't set by default!)

Then uninstall Laravel 5, using add remove programs. Only Laragon will be removed, the www directory will remain

Rename the laragon folder: e.g. c:\laragon to c:\laragon_old

Reinstall [How to set up Laragon on a new Windows computer (part 1) - Install Laragon](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/)

This way I can start with a clean install. I can copy the projects I'm still working on back in and only restore the databases I need.

**Comment by pacmanito** [comment]
@leokhoa can you please share the information related to update to 6.0 which was on forums? AFAIK you adviced to just copy new files for 5.0, is it the same now? It's a pity all the information from forums is gone BTW, as there were many handy solutions there.

**Reply by Sheldon-Monk** [reply]
Similar question:  In which tabs will we find the latest announcements of latest updates??  Thanks in advance.

**Comment by dennist12** [comment]
Download the new release(laragon.exe) and paste to laragon(C:\laragon) folder

**Comment by KoolPal** [comment]
@leokhoa Some help here please!
